react-ai-chatbot/src/components/Controls/Controls.module.css

```
@@ -36,11 +37,14 @@
     border-radius: 50%;
     outline: none;
     background: #0d0d0d;
+    background: light-dark(#0d0d0d, #ececec);
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

background is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)